lib/resources/qubic_js.dart

```diff
@@ -258,6 +258,7 @@ class QubicJs {
           .signUTF8GenericError(result.error ?? ""));
     }
     final Map<String, dynamic> data = json.decode(result.value);
+    QubicSignResult res = QubicSignResult.fromJson(data);
     return QubicSignResult.fromJson(data);
```

Comment:
is this duplication needed? shouldn't the return line return the res variable?

Reply:
Yes, missed linter's message. Fixed